### DISCUSSION TITLE
Persistent Volume Extending Instruction

### DISCUSSION THREAD

**Discussion by themkarimi** [strimzi/strimzi-kafka-operator#7682] [Q&A]
**Suggestion / Problem**
We tried to extend the persistent volume of our Kafka Brokers according to the documentation provides in this [link](https://strimzi.io/blog/2019/02/28/resizing-persistent-volumes/) , although the cluster operator was able to increase the Persistent Volumes but the result of df -h command in the broker container did not show any change. after  bringing down the cluster and increasing the brokers volume size again  we were able to see the brokers filesystem extended as well.

**Documentation Link**
https://strimzi.io/blog/2019/02/28/resizing-persistent-volumes/

**Comment by mojtaba-naseri** [comment]
Your documents helped me solve this problem

**Comment by scholzj** [comment]
The volume resizing works differently on different infrastructure, environment and cloud providers. It is not really possible to provide a single guide for everyone.
